app/backend/resources/languages.json

```diff
@@ -0,0 +1,706 @@
+[
```

Comment:
Does it make sense for this .json file to live in the backend-only folder? I can imagine that frontend might also want to use this, and in that case it would be annoying if you have two separate sources of truth for allowed languages/countries.

Reply:
There's an RPC (API method) for frontend to get it. Better to have one Source of Truth (the backend) that will then serve it to any frontend that needs it rather than syncing across all frontends (web,ios,android) + backend

Reply:
I agree that the frontend should be able to compile this list in, and only optionally use the RPC method for convenience. (We have no versioning in place anyway)

Reply:
My point is not that having the RPC is bad, but that resources which are not intrinsically backend-only should preferably live in a folder that is not exclusive to backend. We can still host it exclusively via RPC first from a project-root based resources directory, but by hosting it from a shared folder it is visible for FE (or any other branch of the project) so they know they compile it in when that makes more sense.